js/ui/components/clubhouse.js

```diff
@@ -251,10 +251,15 @@ var ClubhouseAnimator = new Lang.Class({
         let realUri = this._getClubhouseUri(uri);
         let metadata = this._animations[uri];
 
-        return new Animation(Gio.File.new_for_path(realUri),
-                             metadata.width,
-                             metadata.height,
-                             200);
+        let animation = new Animation(Gio.File.new_for_path(realUri),
+                                      metadata.width,
+                                      metadata.height,
+                                      100);
```

Comment:
Does it matter if we put `100` or `200` here at this point?

Reply:
Nope (assuming we have a delays array in the metadata). But I didn't want to change the constructor. I can add a default value to that arg though, if you prefer.

Reply:
No worries, I just didn't want to change it from one arbitrary value to another :-) I just left it at `200` here.